src/usdb_syncer/hooks.py

```diff
@@ -37,7 +40,13 @@ def unsubscribe(cls, func: Callable[P, None]) -> None:
     @classmethod
     def call(cls, *args: P.args, **kwargs: P.kwargs) -> None:
         for func in cls._subscribers:
-            func(*args, **kwargs)
+            try:
+                func(*args, **kwargs)
+            except Exception as e:  # pylint: disable=broad-except
+                logger.debug(e, exc_info=True)
+                logger.warning(
+                    f"Plugin error in {func.__name__}: {e.__class__.__name__}"
```

Comment:
Here, I would think the exception message is more friendly / useful than its class name.